buildSrc/version.properties

```diff
@@ -20,7 +20,7 @@ httpclient        = 4.5.2
 httpcore          = 4.4.5
 # When updating httpasyncclient, please also update core/src/main/resources/org/elasticsearch/bootstrap/test-framework.policy
 httpasyncclient   = 4.1.2
-commonslogging    = 1.1.3
+commonslogging    = 1.2
```

Comment:
I think we need to look at what modules have a dependency on commonslogging and see if we can change it across the board like this. This also impacts downstream projects like plugins that rely on the version properties here in core to determine the version that they use.